src/main/java/dev/sefiraat/cultivation/implementation/listeners/CustomPlacementListener.java

```diff
@@ -112,6 +112,7 @@ private void unsafelyKillItem(@Nonnull Location location, @Nullable SlimefunItem
             location.getWorld().dropItem(location, bush.getItem().clone());
             bush.removeBushDisplayGroup(location);
             BlockStorage.clearBlockInfo(location);
+            location.getBlock().setType(Material.AIR);
         }
```

Comment:
I've found bushes have similar issues as plants in regards to the `unsafelyKillItem` method. To fix this, I've added:
 
```
} else if (slimefunItem instanceof CultivationBush bush) {
            location.getWorld().dropItem(location, bush.getItem().clone());
            bush.removeBushDisplayGroup(location);
            BlockStorage.clearBlockInfo(location);
            location.getBlock().setType(Material.AIR);
        }
```
Like most things, I'm not 100% this is efficient, but it's been working for us for a while

Reply:
?

Reply:
Ignore me, its been a long day :D I was thinking of something else that's not in scope here, apologies!

For reference I was thinking about https://github.com/Sefiraat/Cultivation/blob/b1dfa47422762d1589d8301403d95fedf9301de1/src/main/java/dev/sefiraat/cultivation/api/slimefun/items/plants/CultivationPlant.java#L155 where I added `event.getBlock().setType(Material.AIR);` to stop a dupe